### PR TITLE
Feat: Improve chat page UI & Add animation to logo

### PR DESCRIPTION
**Pull request by dipasukhaedbert**

### CODE REVIEW

File: src/components/chat-info/chat-info.component.jsx [hunk 1]
```diff
@@ -3,72 +3,46 @@ import "./chat-info.styles.css";
...
-
-  useEffect(() => {
-    db.collection(`/user/${friends}/private_info/`)
+    db.collection(`/user/${props.friendName}/private_info/`)
       .get()
       .then((data) => {
         data.forEach((doc) => {
           setMail(doc.data().email);
         });
       });
```
Comment (felixhalim) [inline src/components/chat-info/chat-info.component.jsx:40]: code repetition

File: src/components/chat-info/chat-info.component.jsx [hunk 2]
```diff
@@ -109,11 +83,16 @@ const ChatInfo = () => {
           <p className="world">{world}</p>
         </div>
         <div className="telegram--icon">
-          <img
-            id="telegram--size"
-            src="https://i.pinimg.com/originals/99/f0/3f/99f03fdee90d871d3d1e718c82feb8be.png"
-            alt="telegram--icon"
-          />
+          <a
+            href={`https://web.telegram.org/#/im?p=@${teleID}`}
```
Comment (felixhalim) [inline src/components/chat-info/chat-info.component.jsx:87]: change to "https://t.me/$(username)"

File: src/components/chat-particulars/chat-particulars.component.jsx [hunk 3]
```diff
@@ -3,58 +3,30 @@ import "./chat-particulars.styles.css";
...
-  }, [friends, friendName]);
-
-  const getFriendBio = () => {
-    db.doc(`/user/${friends}`)
+    db.doc(`/user/${props.friendName}`)
       .get()
       .then((doc) => {
         setFriendBio(doc.data().bio);
       });
   };
```
Comment (felixhalim) [inline src/components/chat-particulars/chat-particulars.component.jsx:27]: code repetition

File: src/components/chat-info/chat-info.component.jsx (hunk 2 above)
Comment (dipasukhaedbert) [reply src/components/chat-info/chat-info.component.jsx:87]: Got it

File: src/components/chat-info/chat-info.component.jsx (hunk 1 above)
Comment (dipasukhaedbert) [reply src/components/chat-info/chat-info.component.jsx:40]: Noted and solved

File: src/components/chat-particulars/chat-particulars.component.jsx (hunk 3 above)
Comment (dipasukhaedbert) [reply src/components/chat-particulars/chat-particulars.component.jsx:27]: Noted and solved.

**Pull request merged**